WordPress/WordPressTest/PromptRemindersSchedulerTests.swift

```diff
@@ -411,8 +411,8 @@ private extension PromptRemindersSchedulerTests {
                 "date": Self.dateFormatter.string(from: date),
                 "answered": false,
                 "answered_users_count": 0,
-                "answered_users_sample": []
-            ])
+                "answered_users_sample": [String: Any]()
+            ] as [String: Any])
```

Comment:
I dismissed my review because I noticed many CI failures. I checked this out locally and got similar results, including a crash in this test:

<img width="1304" alt="Screenshot 2023-04-14 at 11 57 17 am" src="https://user-images.githubusercontent.com/1218433/231922435-f2b14c26-2de5-401e-a58b-483c5fd68aac.png">


For context, I was hoping to merge this to reduce the noise and investigate a different test failure.

Reply:
Looks like the type expected here was `Array<Dictionary<String: Any>>` not `Dictionary<String: Any>`.

Reply:
See https://github.com/wordpress-mobile/WordPress-iOS/pull/20542.

Reply:
Thanks for the investigation and fix!